packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -17,6 +17,11 @@ export class LocalChannelStorageService implements IChannelStorageService {
         return contents !== undefined ? Promise.resolve(contents) : Promise.reject(new Error("Not found"));
     }
 
+    public async readString(path: string): Promise<string> {
+        const contents = this.readStringSync(path);
+        return contents !== undefined ? Promise.resolve(contents) : Promise.reject(new Error("Not found"));
```

Comment:
The code will be more readable if you do the following:
```
assert(contents !== undefined, "Not Found");
return contents;
```
As a general rule, try to throw (not return rejected promises), and return actual values (no need to wrap them into resolved Promise - JS engine can do it for you).